bootstrap/bootstrap.go

```diff
@@ -202,6 +202,11 @@ func startServer(
 		return fmt.Errorf("failed to create EVM requester: %w", err)
 	}
 
+	startingEVMHeight, err := blocks.LatestEVMHeight()
```

Comment:
I don't think the latest EVM height at the server restart should be used anywhere. We always resume syncing blocks from the latest block in the database or if database is empty we use the config init height value.